content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,233 @@
+========
+Chatbots
+========
+
+A *chatbot* is a program designed to mimic a conversation with a live human. Chatbots are assigned
+a script of pre-written steps to follow. These scripts are designed to anticipate a visitor's
+potential response, and lead them through a series of questions and answers the same way a live
+team member would. Chatbots can be customized to fill various roles, from customer support, to
+creating leads, to collecting contact information. The goal of the chatbot will depend on the page
+of the website they are assigned, and the messages included in the script, among other criteria.
+
+.. image:: chatbot/chatbot-visitor-view.png
+   :align: center
+   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+Build a chatbot
+===============
+
+Before creating a new chatbot, the *Live Chat* application must first be installed on the database.
+This can be done directly from the :menuselection:`Apps` menu by searching for :guilabel:`Live
+Chat` in the :guilabel:`Search bar` and clicking :guilabel:`Install`.
```

Comment:
I discussed this with @samueljlieber, since I am directing them to enter something in the search bar, it is part of the directions and, it should have a guilabel. 
I will update the format for Live Chat (with the backticks) 👍